### PR TITLE
chore(gh-actions): install conntrack before minikube

### PR DESCRIPTION
**Pull request by njhale**
Get minikube action working by installing its missing conntrack dependency.

### CODE REVIEW

**Comment by awgreene** [comment]
`level=error msg="Error: error updating S3 Bucket (terraform-20200415132127375000000001) tags: error listing resource tags (terraform-20200415132127375000000001): NoSuchBucket: The specified bucket does not exist"`
/retest

File: .github/workflows/test-scripts.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - run: |
+        sudo apt-get install conntrack
```
Comment (benluddy) [inline .github/workflows/test-scripts.yml:15]: Does this get stuck on interactive prompts without `-y`?

Comment (njhale) [reply .github/workflows/test-scripts.yml:16]: An example I saw didn't, but to be safe I updated the script to fire when it's changed so we can know for sure.

Comment (benluddy) [reply .github/workflows/test-scripts.yml:16]: Might be covered by the runner VM: https://github.com/actions/virtual-environments/commit/b1026148e82a01ca6f1a9e3157ad806ec825df46

**Comment by njhale** [comment]
Superseded by #1452

**Pull request closed without merging**